examples/all-clusters-app/nrfconnect/main/include/CHIPProjectConfig.h

```diff
@@ -30,4 +30,4 @@
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE 20202021
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
 
-#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 2000_ms32
+#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 1000_ms32
```

Comment:
Why are we lowering this? If anything, for us to validate reasonably sensible SED behaviors, we should making this a bit higher....

Reply:
Yeah, but if we want to use the examples for daily/nightly testing, having a long SED interval can greatly increase the time to execute a test suite. Previous the SED role was disabled by default, so making it enabled even with a short interval still improves the test coverage and is not painful when testing things on desk. How about I expose those intervals via Kconfig so that everyone can set the interval at build time with `-DCONFIG_CHIP_SED_IDLE_INTERVAL=20000` flag?

Reply:
I added the change to expose intervals via Kconfig